containers/ecr-viewer/src/app/view-data/utils/date-utils.ts

```diff
@@ -0,0 +1,118 @@
+import { toSentenceCase } from "@/app/services/formatService";
+
+export interface DateRangePeriod {
+  startDate: Date;
+  endDate: Date;
+}
+
+export enum DateRangeOptions {
+  Last24Hours = "last-24-hours",
+  Last7Days = "last-7-days",
+  Last30Days = "last-30-days",
+  Last3Months = "last-3-months",
+  Last6Months = "last-6-months",
+  LastYear = "last-year",
+}
+
+export const dateRangeLabels: { [key in DateRangeOptions]: string } = {
+  [DateRangeOptions.Last24Hours]: toSentenceCase(
+    DateRangeOptions.Last24Hours.replace(/-/g, " "),
+  ) as string,
+  [DateRangeOptions.Last7Days]: toSentenceCase(
+    DateRangeOptions.Last7Days.replace(/-/g, " "),
+  ) as string,
+  [DateRangeOptions.Last30Days]: toSentenceCase(
+    DateRangeOptions.Last30Days.replace(/-/g, " "),
+  ) as string,
+  [DateRangeOptions.Last3Months]: toSentenceCase(
+    DateRangeOptions.Last3Months.replace(/-/g, " "),
+  ) as string,
+  [DateRangeOptions.Last6Months]: toSentenceCase(
+    DateRangeOptions.Last6Months.replace(/-/g, " "),
+  ) as string,
+  [DateRangeOptions.LastYear]: toSentenceCase(
+    DateRangeOptions.LastYear.replace(/-/g, " "),
+  ) as string,
+};
```

Comment:
```suggestion
export const dateRangeLabels = Object.keys(DateRangeOptions).reduce((res, k) => {
  if (typeof DateRangeOptions[k] === 'number') {
    res[k] = k.toLowerCase()
  }
  return res
}, {}
)
```

(needs some more massaging to convince typescript this is fine)

Reply:
Thank youuu for this, was struggling with this 😮‍💨

Reply:
Uncommitted the change till later, need to figure out why tests are failing w/ this change